README.md

```diff
@@ -1,64 +1,97 @@
-#WFM CLI
-A CLI tool for working with the Raincatcher demos and developing the Raincatcher modules.
+#WFM Command Line Interface (CLI)
 
-## Getting started
-If this is your first time working with Raincatcher, do the following:
+This module provides a CLI tool for working with the Raincatcher demos and developing the Raincatcher modules. The Raincatcher demo solution has a dependency on many separate modules. 
 
-1. Clone this CLI tool:
+It is possible to manually set up your local development environment without this tool, but this tool provides functionality to do the required steps automatically.
+
+This CLI tool allows developers to:
+
+- Clone the demo solution to a local machine.
+- Clone all of the Raincatcher modules that the demo solution requires.
+- Run the full demo solution locally.
+- Set up for local development by linking all of the raincatcher modules to the demo solution.
+
+## Getting Started
+
+To get set up for developing Raincatcher locally it is recommended that you set up a separate folder to contain all of the Raincatcher modules and demo apps. (e.g. `~/raincatcher`).
+
+The following steps will assume that the *WORKING_DIRECTORY* is the folder that you use for your local development.
+
+If this is your first time working with Raincatcher, perform the following steps to set up your local development environment:
+
+1. Change to the *WORKING_DIRECTORY*
```

Comment:
If you change the headers to comments and make `WORKING_DIRECTORY` an actual variable you could turn this part into a copypasteable bash script!

i.e.
```bash
export RAINCATCHER_DIRECTORY="~/raincatcher/"
# 1. Change to the *WORKING_DIRECTORY*
mkdir -p $RAINCATCHER_DIRECTORY
cd $RAINCATCHER_DIRECTORY
git clone...
```
etc.